.github/workflows/ci.yml

```
@@ -0,0 +1,71 @@
+name: CI
+
+on: 
+  pull_request:
+    branches:
+      - master
+  push:
+    branches-ignore:
+      - '**'
```

This ignores all branch builds. We should at least run those on master:

```suggestion
    branches:
      - master
```

---

The idea was to run when there is a pull / mr request against master. This is more robust, though.